### PR TITLE
Updates to include a strong regex for 3 out of 4 character classes

### PR DESCRIPTION
**Pull request by hamishwstewart**
Updates index.js file to include a new RegEx to allow for checking that 3 out 4 character classes have been used

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,20 +1,18 @@
 module.exports = (password) => {
-  if (!password) {
-    throw new Error("Password is empty.");
-  }
 
+  const disallowed = '(?=.*[£"\'\\\\])';
```
Comment (u01jmg3) [inline index.js]: - Should this variable be called `disallowedRegex`?
- Do we mean `\\\\` or `\\`?

File: index.js [hunk 2]
```diff
@@ -1,20 +1,18 @@
 module.exports = (password) => {
-  if (!password) {
-    throw new Error("Password is empty.");
-  }
```
Comment (u01jmg3) [inline index.js:4]: I would have put this in a separate commit

File: index.js [hunk 3]
```diff
@@ -54,17 +52,34 @@ module.exports = (password) => {
...
+  if (extraStrongRegex.test(password)) {
+    strength = {
+      id: 3,
+      value: "Extra Strong",
+    };
+  } else if (strongRegex.test(password)) {
     strength = {
       id: 2,
-      value: "Strong",
+      value: "Strong - Can now submit",
```
Comment (u01jmg3) [inline index.js]: Is "Can now submit" necessary in this library?

File: index.js (hunk 1 above)
Comment (hamishwstewart) [reply index.js]: Yeah, this was quite strange as I had to escape the backslash multiple times to get it to work properly, so the \\\\ is correct.

File: index.js (hunk 3 above)
Comment (hamishwstewart) [reply index.js]: No, I just put it in to provide confirmation to the user that a strong password is good enough for submission. It's not really necessary though.

File: index.js (hunk 2 above)
Comment (hamishwstewart) [reply index.js:4]: Noted, but I thought making all the required changes in one commit would be acceptable, as there's not too many of them?

File: index.js (hunk 3 above)
Comment (u01jmg3) [reply index.js]: I would remove this then

File: index.js (hunk 1 above)
Comment (u01jmg3) [reply index.js]: > _Should this variable be called `disallowedRegex`?_

Comment (hamishwstewart) [reply index.js]: Yes. I have updated this now.

File: index.js (hunk 3 above)
Comment (hamishwstewart) [reply index.js]: Removed additional text.

**Review by u01jmg3 — APPROVED** [review]
